sample/HttpTrigger-Disabled/function.json

```diff
@@ -1,5 +1,4 @@
 ﻿{
-    "disabled": true,
```

Comment:
For the test relying on this, the fixture now sets the requisite AzureWebJobs.HttpTrigger-Disabled.Disabled env var to disable the function